aiogram/dispatcher/filters/builtin.py

```diff
@@ -50,6 +51,19 @@ def __init__(self, commands: Union[Iterable, str],
         self.prefixes = prefixes
         self.ignore_case = ignore_case
         self.ignore_mention = ignore_mention
+        for command in self.commands:
+            if not isinstance(self.prefixes, str):
+                for prefix in self.prefixes:
+                    if command.startswith(prefix):
+                        warn(
+                            f'command "{command}" starts with command prefix "{prefix}", so handler will trigger '
+                            f'only on "{prefix + command}" command. Remove leading command prefix to avoid this '
+                            f'behavior.', UserWarning)
+            else:
+                if command.startswith(self.prefixes):
+                    warn(f'command "{command}" starts with command prefix "{self.prefixes}", so handler will trigger '
+                         f'only on "{self.prefixes + command}" command. Remove leading command prefix to avoid this '
+                         f'behavior.', UserWarning)
```

Comment:
Thanks for review.
Did some refactor. Is it OK?

Reply:
Is not OK in because `self.prefixes` can be string with multiple prefixes (see below place where prefixes is checked).

For example `prefixes='/!'` is valid argument and that's mean command can be prefixed by `/` or `!`. So please undo you'r last commit. Sorry for that.

About all other i must test it soon or you can add tests for this place and that can make approving faster.